src/components/MovieDetails/MovieDetails.stories.jsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable */
+import MovieDetails      from './MovieDetails';
+import {Container, Grid} from "@mui/material";
+import headerStyles      from "../Header/Header.module.css";
+import appStyles         from "../../App.module.css";
+
+export default {
+    title     : "Movies/MovieDetails",
+    component : MovieDetails,
+    parameters: {
+        backgrounds: {
+            default: 'content'
+        }
+    }
+};
+
+const Template  = args => (
+        <Container className={appStyles.app} fixed>
+            <Grid xs={12} container className={headerStyles.appHeader} item={true}>
+                <MovieDetails {...args} />
+            </Grid>
+        </Container>
+)
+
+export const Default = Template.bind({})
+Default.story        = {
+    name: 'Short Overview',
+    args: {
+        info: {
+            "id"         : 181808,
+            "title"      : "Star Wars: The Last Jedi",
+            "tagline"    : "The Saga Continues",
+            "voteAverage": 7.1,
+            "vote_count" : 4732,
+            "releaseYear": '2017',
+            "imageUrl"   : "https://image.tmdb.org/t/p/w500/kOVEVeg59E0wsnXmF9nrh6OmWII.jpg",
+            "overview"   : "Rey develops her newly discovered abilities with the guidance of Luke Skywalker, who is unsettled by the strength of her powers. Meanwhile, the Resistance prepares to do battle with the First Order.",
+            "budget"     : 200000000,
+            "revenue"    : 1325937250,
+            "genres"     : [
+                "Fantasy",
+                "Adventure",
+                "Science Fiction"
+            ],
+            "runtime"    : '2h 32min'
+        }

```

Comment:
no need to write extra commas for keys in object and how about to extract this object in constant.js and reuse in test and in story
```suggestion
info: {
            id: 181808,
            title: "Star Wars: The Last Jedi",
            tagline: "The Saga Continues",
            voteAverage: 7.1,
            vote_count: 4732,
            releaseYear: '2017',
            imageUrl: "https://image.tmdb.org/t/p/w500/kOVEVeg59E0wsnXmF9nrh6OmWII.jpg",
            overview: "Rey develops her newly discovered abilities with the guidance of Luke Skywalker, who is unsettled by the strength of her powers. Meanwhile, the Resistance prepares to do battle with the First Order.",
            budget: 200000000,
            revenue: 1325937250,
            genres: [
                "Fantasy",
                "Adventure",
                "Science Fiction"
            ],
            runtime: '2h 32min'
        }
```